mkosi/bootloader.py

```diff
@@ -501,17 +501,63 @@ def sign_efi_binary(context: Context, input: Path, output: Path) -> Path:
     assert context.config.secure_boot_key
     assert context.config.secure_boot_certificate
 
-    if (
+    sbsign = context.config.find_binary("systemd-sbsign", "/usr/lib/systemd/systemd-sbsign")
+    if context.config.secure_boot_sign_tool == SecureBootSignTool.systemd and not sbsign:
+        die("Could not find systemd-sbsign")
+
+    cmd: list[PathString]
+    options: list[PathString]
+
+    if context.config.secure_boot_sign_tool == SecureBootSignTool.systemd or (
+        context.config.secure_boot_sign_tool == SecureBootSignTool.auto and sbsign
```

Comment:
Haven't checked this, but
```suggestion
        context.config.secure_boot_sign_tool == SecureBootSignTool.auto and sbsign is not None
```
should make the assert below unnecessary and would be more symmetric with the predicate in the other branch.

Reply:
Doesn't make the assert unnecessary unfortunately